x/superfluid/keeper/epoch.go

```diff
@@ -61,7 +63,7 @@ func (k Keeper) AfterEpochStartBeginBlock(ctx sdk.Context) {
 }
 
 func (k Keeper) MoveSuperfluidDelegationRewardToGauges(ctx sdk.Context, accs []types.SuperfluidIntermediaryAccount) {
-	bondDenom := k.sk.BondDenom(ctx)
+	bondDenom := k.sk.GetParams(ctx).BondDenom
```

Comment:
nit: we should really cache the BondDenom, otherwise we are unmarshalling all the staking keeper params just for the denom

Reply:
Alternatively, we could just set these to the BaseUnit string here as well, never really understood why these needed to be pulled from the staking params

Reply:
yeah, I think we should make one PR where we change this globally. The problem is that some module have uosmo hardcoded (protorev) and some tests assume stake. I tried changing it, but changing it in one place would break it in another and fixing that one would break the next and so on